alts/src/test/java/io/grpc/alts/internal/AltsAuthContextTest.java

```diff
@@ -36,11 +40,15 @@
   private static final String TEST_PEER_SERVICE_ACCOUNT = "peer@gserviceaccount.com";
   private static final String TEST_RECORD_PROTOCOL = "ALTSRP_GCM_AES128";
 
+  private Map<String, String> testPeerAttributes = new HashMap<String, String>();
+  
```

Comment:
nit: empty line

Reply:
will fix!

Reply:
Is this empty line removed? 
Also, does it make sense to not initializing the map here, but moving the initialization to the place where we do SetUp()? That might make the code cleaner IMO.

Reply:
Yeah that makes sense. I believe that was a fragment from when I had the map along with the static variables but good catch